hub/package.json

```diff
@@ -46,9 +47,8 @@
     "start": "npm run build && node lib/index.js",
     "build": "npm run lint && babel src -d lib && chmod +x lib/index.js",
     "flow": "flow",
-    "lint": "eslint src",
-    "test-inner": "npm run build && npm run flow && babel ./test/src -d ./test/lib/ && node ./test/lib/index.js",
-    "test": "nyc --reporter=text npm run test-inner"
+    "lint": "eslint src && eslint test/src",
+    "test": "npm run flow && NODE_ENV=test nyc --reporter=text node ./test/src/index.js"
```

Comment:
We should add the linter back in to the test command here. `npm run start` fails on this branch.